mobile/src/features/Discover/useCases/SelectDappFromList/DAppListItem/DAppListItem.tsx

```diff
@@ -238,7 +241,7 @@ const SingleAddressDAppWarning = () => {
       content={
```

Comment:
Missing Modal.Content

Reply:
is already under Modal.content  - line 118